### DISCUSSION TITLE
b-form-radio-group not works with validations

### DISCUSSION THREAD

**Discussion by greg-vstorm** [bootstrap-vue/bootstrap-vue#6667] [Q&A] [answered]
I had issue with `b-form-radio-group` and validation libraries like vee-validate, vuelidate. Example code https://codesandbox.io/s/bootstrapvue-vuelidate-example-forked-72mrq?file=/App.vue

```html
<ValidationProvider
    v-slot="validationContext"
    rules="required"
    name="Gender"
  >
    <b-form-group
      v-slot="{ ariaDescribedby }"
      label="Gender"
      :state="validationContext.errors[0] ? false : null"
      :invalid-feedback="validationContext.errors[0]"
    >
      <b-form-radio-group
        id="gender-group-1"
        v-model="gender"
        :options="genderOptions"
        :aria-describedby="ariaDescribedby"
        name="gender"
        stacked
      />
      gender value: "{{ gender }}"
    </b-form-group>
  </ValidationProvider>
```

**Comment by greg-vstorm** [comment, marked as answer]
**[Solution](https://github.com/bootstrap-vue/bootstrap-vue/issues/6666)**, thanks @Hiws is just remove ` v-slot="{ ariaDescribedby }"`  and usage `:aria-describedby="ariaDescribedby"`. All should works fine now.
I write this for the future for someone who make mistake like mine.

**Reply by Hiws** [reply]
I'm not 100% sure where is the issue lies. Whether it's `<b-form-group>` that breaks it, or `<validation-provider>` when used along-side a scoped default slot.
